cli/cmd/root.go

```diff
@@ -47,6 +47,7 @@ var (
 	apiAddr               string // An empty value means "use the Kubernetes configuration"
 	kubeconfigPath        string
 	kubeContext           string
+	kubeNamespace         string
```

Comment:
would it be more accurate to name this variable `defaultNamespace`? perhaps with a comment explaining that it is taken from the kube context?

Reply:
I was a bit reluctant to change it into `defaultNamespace` although it does make more contextual sense. 

There is already a `defaultNamespace` constant for the default control-plane namespace. I've changed that to be `defaultLinkerdNamespace` though, and then changed `kubeNamespace` to be `defaultNamespace` as suggested!